.github/workflows/ulwgl-python.yml

```diff
@@ -32,5 +33,10 @@ jobs:
         pip install ruff
         ruff --output-format github ulwgl_*.py
     - name: Test with unittest
+      if: matrix.python-version == 3.10
+      run: python3 ulwgl_test.py
+    - name: Test with unittest
+      if: matrix.python-version == 3.11 || matrix.python-version == 3.12
```

Comment:
Same here. Maybe unconditionally run `ulwgl_tests` above for all versions, and run `ulwgl_tests_plugins` here only, for 3.11 and 3.12